platform-core-module/src/main/java/com/softicar/platform/core/module/page/PageHeaderAndContentDiv.java

```diff
@@ -32,10 +32,10 @@ public void openPage(PageNavigationEntry<?> linkEntry) {
 
 		removeChildren();
 		CurrentDomPopupCompositor.get().closeAll();
-
 		CurrentDomDocument.get().getDeferredInitializationController().clear();
-
 		DbTableRowCaches.invalidateAll();
+		System.gc();
```

Comment:
This might cut both ways: We could even introduce delays where there are currently none. Plus, it's non-deterministic.
But we can try this, for sure.

Reply:
tested this locally, no difference